doc/nrf/drivers/wifi/nrf700x/nrf700x.rst

```diff
@@ -71,6 +71,14 @@ The nRF7002 DK uses QSPI whereas the nRF7002 EK uses SPI.
 
 To connect the nRF7002 EK to the SoC, the ``nrf7002ek`` shield is required.
 
+Configuration
+*************
+
+The nRF700X Wi-Fi driver has the following configurable options:
```

Comment:
```suggestion
The nRF700X Wi-Fi driver has the following configuration options:
```

Reply:
Can you also change line 58 to Radio **T**est mode for consistency (I cannot comment there)?

Reply:
Done.